.gitignore

```diff
@@ -12,3 +12,5 @@ npm-debug.log
 
 # Firebase
 firebase-debug.log
+
+.idea/
```

Comment:
Actually I only wanted to make a pull request for "More flexible icons loader", not the merge commit and this one... I was all added automatically when I pushed to my branch. I will try to revert.